src/Components/Web/ref/Microsoft.AspNetCore.Components.Web.csproj

```diff
@@ -8,11 +8,13 @@
     <Reference Include="Microsoft.AspNetCore.Components"  />
     <Reference Include="Microsoft.AspNetCore.Components.Forms"  />
     <Reference Include="Microsoft.Extensions.DependencyInjection"  />
+    <Reference Include="Microsoft.JSInterop"  />
```

Comment:
Can you remind me specifically? What part of `.Web` requires interop?

Reply:
Nothing directly. Mvc.ViewFeatures, Components.Server, and Blazor all pick it up from here and `.Web` seemed like the right layer for this API. But we could move it further up the stack. What do you think?

Reply:
Where does `ViewFeatures` use it?

I'm of two mins about this.
1. JS interop is a definitely feature of *web* (logically). If we needed to add a component to *web* that uses JSInterop, we wouldn't hesitate.
2. It's always good to push on these things and see what falls out. You can always move things lower in the stack later because it's not a breaking change.

Reply:
ViewFeatures uses it for UnsupportedJSRuntime. I'm going to move this further "up" the stack in the merge PR.

Reply:
Moving it up is a bit broken for component libraries. They fail to find a reference to `IJSRuntime`. If we want to be super clean about this, we could consider creating JSInterop.Abstraction which Web references and the implementation is referenced by Blazor \ Server. Let's talk about this in the stand up. For now leaving this in Web

Reply:
I don't think there's any time left to do something like that, even if it were a good idea 😢